integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/EDUCoreConverter.java

```diff
@@ -17,7 +17,8 @@
         try {
             jaxbContext = JAXBContext.newInstance(EDUCore.class);
         } catch (JAXBException e) {
-            throw new RuntimeException("Could not create JAXBContext for " + EDUCore.class);
+//            throw new RuntimeException("Could not create JAXBContext for " + EDUCore.class, e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)